### PR TITLE
fix: fix image build bug and add some note

### PR DESCRIPTION
**Pull request by carasue**
Change-Id: I10d1cc3d1b4b38805d6db85eed7eb0e4a6f251a3

### CODE REVIEW

**Comment by Shpota** [comment]
Thank you. I'll review the changes shortly.

File: templates/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
 FROM node:12.16 AS JS_BUILD
 COPY webapp /webapp
-WORKDIR webapp
+WORKDIR /webapp
 RUN npm install && npm run build --prod
 
 FROM golang:1.13.8-alpine AS GO_BUILD
-RUN apk add build-base
+RUN  apk update && apk add --virtual build-dependencies build-base gcc wget git
```
Comment (Shpota) [inline templates/Dockerfile]: Thank you, `apk update` is definately missing. But why dobyou need to add more packages in addition to `build-base`?

File: transform/transformer.go [hunk 2]
```diff
@@ -1,6 +1,10 @@
 // Transforms static content form the 'templates' folder
 // in the root of the repository to Go code in order to
 // distribute it as a part of the module.
+
+//Noted: run this script in the transform dir after u changed the templates content
```
Comment (Shpota) [inline transform/transformer.go]: I would omit usage of casual English in documentation. Let's remove `Noted:` and rephrase the wording in order to sound more like a documentation.

File: templates/Dockerfile (hunk 1 above)
Comment (carasue) [reply templates/Dockerfile]: I will fix that, they are redundant

File: transform/transformer.go (hunk 2 above)
Comment (carasue) [reply transform/transformer.go]: I got u.

**Pull request closed without merging**